src/development/LocalTest/appsettings.json

```diff
@@ -36,8 +36,8 @@
   "AllowedHosts": "*",
   "LocalPlatformSettings": {
     "LocalTestingStorageBasePath": "c:/AltinnPlatformLocal/",
-    "AppRepositoryBasePath": "<Path to your repos folder where you have cloned the Altinn Studio application repositories>",
-    "LocalTestingStaticTestDataPath": "<Path to altinn-studio repo>/src/development/TestData/"
+    "AppRepositoryBasePath": "c:/repos",
```

Comment:
Is "c:/" resolvable on a unix environment such as MacOS?
And what differentiates this directory form the c:/AltinnCore/Repos path?

Reply:
this should not be part of it. will be removed. It is my local setup